src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE DATABASE `books_schema`;
```

Comment:
I'd rename this to something else, because schema and a database are not the same things. In MySQL they are not so different as far as I am aware, but in other languages, it might be more significant. Anyway, it would be more correct to name things with what they really are, like `books_db` for example

Reply:
Done